contrib/src/lib.rs

```diff
@@ -74,3 +74,9 @@ mod uuid;
 
 #[cfg(feature = "uuid")]
 pub use uuid::{UUID, UuidParseError};
+
+#[cfg(feature = "handlebars_templates")]
+pub extern crate handlebars;
```

Comment:
Move these `extern crate`s to the top of the file below the existing `extern crate`s.